visualdl/logic/sdk_test.cc

```diff
@@ -217,9 +180,9 @@ TEST(Audio, add_sample_test) {
   for (int step = 0; step < num_steps; step++) {
     audio.StartSampling();
     for (int i = 0; i < 7; i++) {
-      vector<float> data;
-      for (int j = 0; j < 3 * 5 * 5; j++) {
-        data.push_back(float(rand()) / RAND_MAX);
+      vector<uint8_t> data;
```

Comment:
Also here 😃

Reply:
The audio data we accept is vector of uint8_t type, so I randomly pick numbers from rand() % 256 to create a vector